### PR TITLE
Save source cloning stage

### PR DESCRIPTION
**Pull request by dragonfax**
### References

* JIRA https://zendesk.atlassian.net/browse/TOOLS-873

### Risks

None

### CODE REVIEW

File: db/migrate/20160927223306_save_source_template_to_clone_stage.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class SaveSourceTemplateToCloneStage < ActiveRecord::Migration[5.0]
+  def change
+    add_column :stages, :template_stage_id, :integer
```
Comment (grosser) [inline db/migrate/20160927223306_save_source_template_to_clone_stage.rb]: add an index since we query with it ?

**Comment by grosser** [comment]
👍  ... the merge logic can then use that column ?

File: db/migrate/20160927223306_save_source_template_to_clone_stage.rb (hunk 1 above)
Comment (dragonfax) [inline db/migrate/20160927223306_save_source_template_to_clone_stage.rb]: I'm not sure we'll use it, but its harmless to add one.

Comment (dragonfax) [inline db/migrate/20160927223306_save_source_template_to_clone_stage.rb]: This really is just to show the OPS person where the clone came from. As a sanity check. We'll show it in a couple of the view templates showing stages.

**Pull request merged**